js/build.js

```diff
@@ -280,6 +280,10 @@
         var eventDetail = event.detail;
 
         if (eventDetail && eventDetail.type === 'colorChange') {
+          if (eventDetail.widgetId && eventDetail.widgetId !== chartId) {
```

Comment:
I think this will then also ensure that if `widgetId` is missing, do nothing.

```suggestion
          if (!eventDetail.widgetId || eventDetail.widgetId !== chartId) {
```

Does my suggestion seem right to you?

Reply:
Not at all, I thought we need to apply general appearance settings for all charts when `widgetId` is not specified (when we set color for charts in appearance settings without clicking on brush icon).

Reply:
Thanks for explaining